pkg/controller/fedvolumebackup/fed_volume_backup_control.go

```diff
@@ -130,6 +131,17 @@ func (c *defaultBackupControl) updateVolumeBackupMetrics(volumeBackup *v1alpha1.
 	}
 }
 
+// add zero to the backup status counters; this sets the backup counter metric to zero when the federated manager restarts.
+func (c *defaultBackupControl) addZeroToBackupStatusMetric(volumeBackup *v1alpha1.VolumeBackup) {
```

Comment:
nit: s/addZeroToBackupStatusMetric/initBackupStatusMetrics. or reset.

Reply:
I think `initBackupStatusMetrics` is better, we should only do initialization.

Reply:
done